packages/tsdocs/package.json

```diff
@@ -13,7 +13,8 @@
     "extract-apidocs": "node bin/extract-apis",
     "document-apidocs": "api-documenter markdown -i ../../docs/apidocs/models -o ../../docs/site/apidocs",
     "update-apidocs": "node bin/update-apidocs",
-    "build": "lb-tsc && npm run build --prefix fixtures/monorepo",
+    "build": "lb-tsc",
+    "build:fixtures": "npm run build --prefix fixtures/monorepo",
     "clean": "lb-clean loopback-tsdocs*.tgz dist *.tsbuildinfo package",
     "pretest": "npm run build",
```

Comment:
Let's add `npm run build:fixtures` to `pretest`.

Reply:
I don't like the idea of rebuilding fixtures before every test run. I want the tests to finish as quickly as possible, adding a build step that's a no-op in most times is not optimal.